app/main/routes.py

```diff
@@ -84,14 +84,64 @@ def callback():
     session["token_scope"] = access_token_response["scope"]
     session["session_state"] = access_token_response["session_state"]
 
-    return redirect(url_for("main.poc_search"))
+    return redirect(url_for("main.browse"))
```

Comment:
I configured it so that only our non e2e tests contribute to our code coverage as our e2e tests should be supplementary to our normal tests

but its not a big deal that this line is not covered in the unit tests.